src/main/java/transportservice/RunPlugin.java

```diff
@@ -25,9 +25,13 @@
 import org.opensearch.common.util.PageCacheRecycler;
 import org.opensearch.discovery.PluginRequest;
 import org.opensearch.discovery.PluginResponse;
+import org.opensearch.index.IndicesModuleNameResponse;
+import org.opensearch.index.IndicesModuleRequest;
+import org.opensearch.index.IndicesModuleResponse;
 import org.opensearch.indices.IndicesModule;
 import org.opensearch.indices.breaker.CircuitBreakerService;
 import org.opensearch.indices.breaker.NoneCircuitBreakerService;
+import org.opensearch.plugins.PluginsOrchestrator;
 import org.opensearch.search.SearchModule;
 import org.opensearch.threadpool.ThreadPool;
 import org.opensearch.transport.*;
```

Comment:
nit: Why do we have a `*` import. 
Can we add a rule to spotless to avoid this?

Reply:
Yeah that's a good idea. I have removed the `*` though. I'm not aware of such rule but will take a look.